extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -26,17 +26,14 @@ t.run('amp-image-slider', {}, function () {
   // #s2 has attribute `disable-hint-reappear` set
   // A huge padding is added to the bottom to allow room for scrolling
   const sliderBody = `
```

Comment:
nit: you get prettier formatting if you add a leading comment.

```suggestion
  const sliderBody = /* HTML */ `
```

Reply:
Oooh, that's a nice tip!